### PR TITLE
add works and about me sections

### PR DESCRIPTION
**Pull request by MarsIncarnate**
This PR adds a portfolio and about me sections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -101,5 +101,129 @@ <h1>
             <rect x="1" y="1" width="40" height="40" rx="20" stroke="#DFE1E6"/>
         </svg>            
     </a>
+    <section id="portfolio">
+        <h1>My Recent<br>Works
+            <div id="underline"></div>
+        </h1>
```
Comment (algerina) [inline index.html]: - [ ] Kindly replace the h1 tag by an h2 or h3.

File: index.html [hunk 2]
```diff
@@ -101,5 +101,129 @@ <h1>
...
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+    </section>
+    <section id="about">
+        <div id="about-heading">
+            <h1>About me</h1>
```
Comment (algerina) [inline index.html]: - [ ] Please consider replacing the h1 by an h2 or h3 tag as the page should have only one h1.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @MarsIncarnate , 

You've done an amazing job fir 👏 💯 👍
There are some issues that you still need to work on to go to the next project but you are almost there! 


## Highlights
  -  Your project looks awesome!
  -  You have meaningful commits and PR description.

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.


You can also consider:
  - [x] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Review by Qoosim — APPROVED** [review]
#   *Status: Approved* :tada: :heavy_check_mark: 

Hi @MarsIncarnate, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-   No linter errors. :heavy_check_mark: 
-   Good README file. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**